examples/manipulating_visualizing/plot_dim_plotting.py

```diff
@@ -0,0 +1,51 @@
+"""
+Controling the contrast of the background when plotting
+=========================================================
+
+The `dim` argument controls the contrast of the background.
+
+*dim* modifies the contrast of this image: dim=0 leaves the image
+unchanged, negative values of *dim* enhance it, and positive values
+decrease it (dim the background).
+"""
+
+# Retrieve the data: the localizer dataset with contrast maps
+
+from nilearn import datasets
+
+localizer_dataset = datasets.fetch_localizer_contrasts(
+    ["left vs right button press"],
+    n_subjects=2,
+    get_anats=True,
+    get_tmaps=True)
+localizer_anat_filename = localizer_dataset.anats[1]
```

Comment:
> We could add the comments/print to explain the 3/4D thing like in the other
> examples.

But that's not the point of this example. I'd rather keep only one idea
per example